src/components/PupilViews/PupilExperience/PupilExperience.view.tsx

```diff
@@ -132,6 +135,12 @@ export const PupilPageContent = ({
           phase={phase as "primary" | "secondary"}
           starterQuizQuestionsArray={starterQuiz ?? []}
           exitQuizQuestionsArray={exitQuiz ?? []}
+          programmeSlug={browseData.programmeSlug}
```

Comment:
lets just pass through the browse data and the page type for a cleaner interface